docs/accessing-the-cluster.md

```diff
@@ -29,31 +23,67 @@ $ kubectl config view
+  kibana-logging is running at https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/kibana-logging
+  kube-dns is running at https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/kube-dns
+  grafana is running at https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/monitoring-grafana
+  heapster is running at https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/monitoring-heapster 
+}
 ```
 
-Using this information you can now issue the following `curl` command to get status information about
-the Elasticsearch logging service.
+#### Manually constructing proxy URLs
+As mentioned above, you use the `kubectl cluster-info` command to retrieve the service's proxy URL. To create proxy URLs that include service endpoints, suffixes, and parameters, you simply append to the service's proxy URL:  
+`http://`*`kubernetes_master_address`*`/`*`service_path`*`/`*`service_name`*`/`*`service_endpoint-suffix-parameter`*   
+##### Examples
+ * To access the Elasticsearch service endpoint `_search?q=user:kimchy`, you would use:   `http://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/elasticsearch-logging/_search?q=user:kimchy`
+
+ * To access the Elasticsearch cluster health information `_cluster/health?pretty=true`, you would use:   `https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/elasticsearch-logging/_cluster/health?pretty=true`  
+ ```
+  {
+	 "cluster_name" : "kubernetes_logging",
+	 "status" : "yellow",
+	 "timed_out" : false,
+	 "number_of_nodes" : 1,
+	 "number_of_data_nodes" : 1,
+	 "active_primary_shards" : 5,
+	 "active_shards" : 5,
+	 "relocating_shards" : 0,
+	 "initializing_shards" : 0,
+	 "unassigned_shards" : 5
+  }
+```
+
+### Accessing cluster information
+You can run `curl` commands or use a web browser to access the information about cluster services. Depending on how secure the information is, you can choose to use basic authentication or token authentication (bearer and insecure). 
+
+#### Using `curl` commands
+Run `curl` commands using the following formats:
+
+ * Basic authentication: `$ curl -k -u` *`username`*`:`*`password`* *`proxy_URL`*
+ * Token authentication: 
+     * Bearer tokens: `curl --insecure -H "Authorization: Bearer` *`access_token`*`"` *`proxy_URL`*
+	 * Insecure: `curl --insecure -H "Authorization: Bearer` *`access_token`*`"` *`proxy_URL`*
+
+For example, to get status information about the Elasticsearch logging service, you would run one of the following commands:
+
+ * Basic authentication:
+`$ curl -k -u admin:4mty0Vl9nNFfwLJz https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/elasticsearch-logging`
+
+ * Token authentication:
+`$ curl -k -H "Authorization: Bearer cvIH2BYtNS85QG0KSLHgl5Oba4YNQOrx" https://104.197.5.247/api/v1beta3/proxy/namespaces/default/services/elasticsearch-logging`
+
```

Comment:
Im not sure but I assumed it wouldnt because my build still shows v1beta1 (i assumed my build didnt pick up the fixes yet). Ill try (and also check to see if there is a new image available for me to install).